storage/rocksdb/rdb_datadic.cc

```diff
@@ -3875,6 +3875,16 @@ Rdb_ddl_manager::safe_find(GL_INDEX_ID gl_index_id) {
   return ret;
 }
 
+const std::string
+Rdb_ddl_manager::get_table_name(const GL_INDEX_ID &gl_index_id) {
+  std::string ret;
+  auto it = m_index_num_to_keydef.find(gl_index_id);
```

Comment:
Could you move this function after Rdb_ddl_manager::find(), so that safe_find()'s description is more accurate. Also, is mysql_rwlock_rdlock(&m_rwlock); not needed here? If so, there should be a comment to indicate it like the find function. However, given the place you're calling get_table_name() also calls safe_find(), I don't believe the lock is held. If you do acquire the lock, then probably rename this function to safe_get_table_name().